app/code/Magento/InventoryInStorePickupQuote/composer.json

```diff
@@ -4,10 +4,12 @@
   "require": {
     "php": "~7.1.3||~7.2.0||~7.3.0",
     "magento/framework": "*",
+    "magento/module-inventory-in-store-pickup": "*",
     "magento/module-inventory-in-store-pickup-api": "*",
     "magento/module-quote": "*",
     "magento/module-inventory-in-store-pickup-shipping-api": "*",
     "magento/module-store": "*",
+    "magento/module-checkout": "*",
```

Comment:
This is REALLY something we would like to avoid.
Could you re-arrange extensions of order flow to remove dependency on `Magento_Checkou` ?

Reply:
Agree. Fixed